packages/core/src/core/chain-of-thought.ts

```
@@ -1267,68 +1236,16 @@ ${availableOutputs
     public async think(
         userQuery: string,
         maxIterations: number = 10
-    ): Promise<void> {
+    ): Promise<void | string> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)